apps/teams-test-app/src/App.tsx

```diff
@@ -68,12 +68,15 @@ import WebStorageAPIs from './components/WebStorageAPIs';
 
 const urlParams = new URLSearchParams(window.location.search);
 
```

Comment:
Can you add a comment outlining how to craft your URL search parameters so they can be succesfully parsed here please?

Reply:
Okay I've added the comment here.